package.json

```diff
@@ -1,92 +1,100 @@
 {
```

Comment:
Please try to avoid reformatting files.

I think the only change here is you've added `ssh-keygen` as a dependency?

Reply:
Yes, I've added ssh-keygen as a dependency by utilizing npm command.
I will edit package.json manually.